ngui/ui/src/components/forms/ConnectCloudAccountForm/ConnectCloudAccountForm.tsx

```diff
@@ -159,55 +160,56 @@ const getAwsRootParameters = (formData) => {
           region_name: formData[AWS_BILLING_BUCKET_FIELD_NAMES.REGION_NAME] || undefined,
           config_scheme: formData[AWS_ROOT_INPUTS_FIELD_NAMES.CONFIG_SCHEME]
         };
+
+  const extraParams = {
+    use_edp_discount: formData[AWS_USE_AWS_EDP_DISCOUNT_FIELD_NAMES.USE_EDP_DISCOUNT],
+    cur_version: Number(formData[AWS_EXPORT_TYPE_FIELD_NAMES.CUR_VERSION]),
+    ...getConfigSchemeParameters()
+  };
+
   return {
     name: formData[DATA_SOURCE_NAME_FIELD_NAME],
     type: AWS_CNR,
     config: {
       access_key_id: formData[AWS_ROOT_CREDENTIALS_FIELD_NAMES.ACCESS_KEY_ID],
       secret_access_key: formData[AWS_ROOT_CREDENTIALS_FIELD_NAMES.SECRET_ACCESS_KEY],
-      use_edp_discount: formData[AWS_USE_AWS_EDP_DISCOUNT_FIELD_NAMES.USE_EDP_DISCOUNT],
-      cur_version: Number(formData[AWS_EXPORT_TYPE_FIELD_NAMES.CUR_VERSION]),
-      ...getConfigSchemeParameters()
+      ...(connectionType !== CONNECTION_TYPES.AWS_MEMBER ? extraParams : {})
```

Comment:
Shouldn't `linked: true` be included when `connectionType !== CONNECTION_TYPES.AWS_MEMBER` (as we do below for `getAwsAssumedRoleParameters`)?

```suggestion
      ...(connectionType !== CONNECTION_TYPES.AWS_MEMBER ? extraParams : { linked: true })
```

It is currently not possible to connect a linked account when selecting Member → Access key, since `linked: true` is not added in this scenario.

Also, the config field should resolve to `awsLinkedConfig`. This should be fixed automatically once we include `linked` in the config.

<img width="1280" height="479" alt="Image" src="https://github.com/user-attachments/assets/e7b0aab4-48ac-4d3c-a761-6eb804b3388f" />